build-scripts/show-manifest

```diff
@@ -0,0 +1,3 @@
+rm -f manifest.xml
+"C:/Program Files (x86)/Windows Kits/10/bin/10.0.19041.0/x86/mt.exe" "-inputresource:$1;#1" "-out:manifest.xml"
```

Comment:
This path is not portable/stable, different versions of the windows 10 kit will have different paths to this.

Reply:
This was a quick & dirty script for me to print the manifest, and not loosing it during a cleanup. I will remove from the repository.